packages/quantic/force-app/main/default/lwc/quanticResultsPerPage/quanticResultsPerPage.html

```diff
@@ -1,9 +1,9 @@
 <template>
-    <template if:true={hasResults}>
-       <div class="slds-float_right">
-            <template for:each={choices} for:item="choice">
-                <c-quantic-number-button key={choice} number={choice} current={currentResultsPerPageValue} ongoto={goto}></c-quantic-number-button>
-            </template>
-        </div> 
-    </template>
+  <template if:true={hasResults}>
+    <div class="slds-float_right">
+      <template for:each={choices} for:item="choice">
+        <c-quantic-number-button key={choice} number={choice} selected={currentResultsPerPageValue} ongoto={goto}></c-quantic-number-button>
```

Comment:
`selected={currentResultsPerPageValue}` won't work. You also need to replace `currentResultsPerPageValue` by a boolean value.

Reply:
I just changed the name, i didn't make it boolean. i could though

Reply:
made it a bool